src/main/java/pers/gwyog/gtneioreplugin/util/DimensionHelper.java

```diff
@@ -94,15 +95,9 @@
             s = s.trim();
             for (int i = 0; i < DimNameDisplayed.length; i++) {
                 if (s.equals(DimNameDisplayed[i])) {
-                    s = DimName[i].replaceAll("GalacticraftCore_", "").replaceAll("GalacticraftMars_", "").replaceAll("GalaxySpace_", "").replaceAll("Vanilla_", "Vanilla ");
-                    if (s.equals("Twilight"))
-                        s = "Twilight Forrest";
-                    else if (s.equals("Underdark"))
-                        s = "Deep Dark";
-                    else if (s.equals("EndAsteroid"))
-                        s = "Far End Asteroids";
-                    
-                    switch (s) {
+                    s = I18n.format("gtnop.world." + DimName[i].replaceAll("GalacticraftCore_", "").replaceAll("GalacticraftMars_", "").replaceAll("GalaxySpace_", "").replaceAll("Vanilla_", "Vanilla "));
```

Comment:
If you move this statement down a line, you can take advantage of the definition of `k` to reduce code duplication here